cpp/memilio/compartments/compartmentalmodel.h

```diff
@@ -129,9 +130,9 @@ struct CompartmentalModel {
      * This can be used as initial conditions in an ODE solver
      * @return the initial populatoins
      */
-    Eigen::VectorXd get_initial_values() const
+    Vector<FP> get_initial_values() const
     {
-        return populations.get_compartments();
+        return populations.get_compartments().template cast<FP>();
```

Comment:
This should not be necessary, right? Shouldn't we, rather than casting, disallow using something other than FP in `Populations`? This feels like it could hide some type errors otherwise.

Reply:
@hannemann-tamas